wikilink/wiki_link.py

```diff
@@ -396,6 +399,48 @@ def _worker(self, execution_queue, storage_queue, separation_queue, event):
 						storage_queue.put(n)
 						separation_queue.put(number_of_sep + 1)
 
+	def _print_path(self):
+		""" 
+			Function prints the sequence of paths from source to destination 
+			with the shortest number of link.
+		Args:
+			None
+		Returns:
+			None
+		"""
+		dest_node = LinkNode(self.dest_id)
+
+		successful_paths = []
+
+		with self._session_scope() as session:
+			def find_route_to_parent(node, source_root_id, depth=0, session=session):

```

Comment:
So what happens if 2 links are parents of each other, i.e. url of 1 can be found from the other wiki page and vice versal; this will create an infinite loop and running out of limit soon

Reply:
We are having check on the depth with limit.